mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -230,6 +231,8 @@ def StartDetectionThread(self, vminitparams, regionname=None, cameranames=None,
             command['destregionname'] = destregionname
         if ignoreBinpickingStateForFirstDetection is not None:
             command['ignoreBinpickingStateForFirstDetection'] = bool(ignoreBinpickingStateForFirstDetection)
+        if clearDetectedCache is not None:
```

Comment:
I am against this kind of None check. This argument should not be allowed to be None.

Reply:
None means you let the defaults decide, i think it is ok....
it is better than having `null` getting through to JSON

Reply:
It's an optional argument. You can just ignore it...